tests/periph_uart/main.c

```diff
@@ -57,16 +57,97 @@ static int parse_dev(char *arg)
 {
     unsigned dev = atoi(arg);
     if (dev >= UART_NUMOF) {
-        printf("Error: Invalid UART_DEV device specified (%u).\n", dev);
+        printf("msg: Invalid UART_DEV device specified (%u).\n", dev);
         return -1;
     }
     else if (UART_DEV(dev) == STDIO_UART_DEV) {
-        printf("Error: The selected UART_DEV(%u) is used for the shell!\n", dev);
+        printf("msg: The selected UART_DEV(%u) used for the shell!\n", dev);
         return -2;
     }
     return dev;
 }
 
+#ifdef MODULE_PERIPH_UART_MODECFG
+static uart_data_bits_t parse_data_bits(char *arg)
+{
+    int data_bits = atoi(arg);
+    uart_data_bits_t res;
+
+    switch (data_bits) {
```

Comment:
here and in the following: how about a lookup tables?!

Reply:
Done